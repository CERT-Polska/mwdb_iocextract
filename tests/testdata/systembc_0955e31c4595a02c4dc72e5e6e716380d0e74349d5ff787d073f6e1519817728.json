{
    "dns": [
        "5.132.191.104",
        "92.163.33.248",
        "206.189.120.27"
    ],
    "host": [
        "107.172.197.105",
        "107.172.197.105"
    ],
    "port": [
        "4016"
    ],
    "type": "systembc"
}