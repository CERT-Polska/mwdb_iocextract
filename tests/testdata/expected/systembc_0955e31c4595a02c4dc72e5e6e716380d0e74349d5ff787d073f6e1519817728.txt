NetLoc 107.172.197.105/
NetLoc 107.172.197.105/
